charts/images.yaml

```diff
@@ -8,7 +8,7 @@ images:
 - name: etcd
   sourceRepository: github.com/gardener/etcd-custom-image
   repository: europe-docker.pkg.dev/gardener-project/public/gardener/etcd
-  tag: "v3.4.26-3"
+  tag: "v3.4.26-7"
```

Comment:
Updating `etcd-custom-image`  to `v3.4.26-7` for druid `v0.22.x` is not correct because `etcd-custom-image v2.4.26-7` contains cert path changes done by this PR: https://github.com/gardener/etcd-custom-image/pull/46
I would suggest you to move `v.3.26-6`.